### ISSUE TITLE
Doesn't work with long content

### ISSUE THREAD

**Issue by PuruVJ** [koka831/remark-custom-container#120]
I have come across an issue where writing

```md
::: note
Depending on the device, you may not see much difference
:::
```

 works perfectly.

<img width="450" alt="CleanShot 2022-11-08 at 11 32 57@2x" src="https://user-images.githubusercontent.com/47742487/200487390-29d720a1-8d99-4cac-b168-bf74485652bf.png">

 but something longer

```md
::: note
If you're programmatically creating the grid, do not set it to \[0, 0\] ever, that will stop drag at all. Set it to `undefined` to make it continuous once again
:::
```

doesn't render. If I delete the 2nd second sentence in it, it renders it properly. Breaking it up in spaces doesn't help either.

<img width="649" alt="CleanShot 2022-11-08 at 11 33 51@2x" src="https://user-images.githubusercontent.com/47742487/200487528-435adc9e-0dbf-4ef0-a084-3bcc8b77f19d.png">

**Comment by talatkuyuk** [comment]
+1

**Comment by talatkuyuk** [comment]
@koka831 You are right to make decision on the line breaks.

Here is an example that I found in the **docusaurus** tutorial
https://tutorial.docusaurus.io/docs/tutorial-basics/markdown-features#admonitions

**Comment by koka831** [comment]
@talatkuyuk Thank you for the helpful links:)
IMHO it makes so convenient to parse custom directive. `unist-util-visit` treats closing `:::` marker as a member of last child inside the directive, when children inside the directive contains HTML or some complex notation.

e.g.)
```markdown
::: info
first child
<div>second child</div>
:::
```

will be converted in (with some simplify):
- `{ type: "paragraph", value: "::: info" }`
- `{ type: "paragraph", value: "first child" }`
- `{ type: "html": value: "<div>second child</div>:::" }` ← this makes hard to parse :sweat_smile: 

And yes, these plugins are not compatible with `micromark` but its source looks nice:)
I'll dig them and consider some refactoring https://github.com/koka831/remark-custom-container/pull/139, thanks!

**Comment by koka831** [comment]
@PuruVJ @talatkuyuk multiple contents support released in https://www.npmjs.com/package/remark-custom-container :tada:
Thank you for reporting the issue and patience again.
If you find something else regarding this release, feel free to report me:)

**Comment by talatkuyuk** [comment]
I tested it. Works like a charm. Thanks.
